src/azul/service/manifest_service.py

```diff
@@ -604,14 +608,15 @@ def entity_type(self) -> str:
     def source_filter(self) -> SourceFilters:
         return [
             *super().source_filter,
-            'contents.files.related_files'
+            'contents.files.related_files',
+            'contents.files.drs_path'
         ]
 
     def write_to(self, output: IO[str]) -> Optional[str]:
         sources = list(self.manifest_config.keys())
         ordered_column_names = [field_name
                                 for source in sources
-                                for field_name in self.manifest_config[source]]
+                                for field_name in self.manifest_config[source]] + ['drs_url']
```

Comment:
We should modify the manifest config instead or explain in a comment why we can't.

Reply:
Never mind. I understand now. Just consider if this could be done more elegantly by modifying the hit instead of the row.

Reply:
I'm not sure if it's appropriate to thumbs-up both comments, since the second seems to cancel the first, but I've attempted to address both issues (updating the manifest config AND avoiding direct modification of the row).